frontend/src/modules/app/ToolBar.tsx

```diff
@@ -132,8 +155,13 @@ const alarmConfiguration = (ventilationMode: VentilationMode | null): Array<Alar
 /**
  * ToolBar
  *
- * A container for displaying buttons that handle system changes based on
+ * @component A container for displaying buttons that handle system changes based on
  * various states and conditions such as ventilator state and current page route.
+ *
+ * @param {React.ReactNode} children Adds dynamic items as children for SidebarSlideRoute
+ * @param {boolean} staticStart staticStart is used as a config for reusing Start/Pause Ventilation button for LandingPage Layout
```

Comment:
What would `staticStart === true` mean? When in the frontend's lifetime does it become true? When in the frontend's lifetime does it become false?

Reply:
staticStart is by default false. It becomes true only on landing page, where we override this to true. Changing it to true does 2 things

1. It will update label of Start/Pause ventilation to 'Loading...' in Landing Page layout as we are waiting until Ventilating
2. It will redirect to to 'Quick Start' Page

Suggestion: I think now we can change the configuration to use 2 different buttons instead of one for simplifying it